tests/test_pfcwd.py

```diff
@@ -0,0 +1,98 @@
+import redis
+import time
+import os
+import pytest
+import re
+import json
+from swsscommon import swsscommon
+
+PFCWD_TABLE_NAME = "DROP_TEST_TABLE"
+PFCWD_TABLE_TYPE = "DROP"
+PFCWD_TC = ["3", "4"]
+PFCWD_RULE_NAME_1 =  "DROP_TEST_RULE_1"
+PFCWD_RULE_NAME_2 =  "DROP_TEST_RULE_2"
+
+class TestPfcWd:
+    def find_acl_rule_oid(self, dvs):
+       self.asic_db = swsscommon.DBConnector(1, dvs.redis_sock, 1)
+       atbl = swsscommon.Table(self.asic_db, "ASIC_STATE:SAI_OBJECT_TYPE_ACL_ENTRY")
+       keys = atbl.getKeys()
+       acl_entry = [k for k in keys if k not in dvs.asicdb.default_acl_entries]
+
+       return acl_entry[0]
+
+    def test_PfcWdAclCreationDeletion(self, dvs, dvs_acl, testlog):
+        try:
+            dvs_acl.create_acl_table(PFCWD_TABLE_NAME, PFCWD_TABLE_TYPE, ["Ethernet0","Ethernet8", "Ethernet16", "Ethernet24"], stage="ingress")
+            dvs.runcmd("counterpoll queue enable")
+            time.sleep(10)
+
+            config_qualifiers = {
+                "TC" : PFCWD_TC[0],
+                "IN_PORTS": "Ethernet0"
+            }
+
+            expected_sai_qualifiers = {
+                "SAI_ACL_ENTRY_ATTR_FIELD_TC" : dvs_acl.get_simple_qualifier_comparator("3&mask:0xff"),
+                "SAI_ACL_ENTRY_ATTR_FIELD_IN_PORTS": dvs_acl.get_port_list_comparator(["Ethernet0"])
+            }
+        
+            dvs_acl.create_acl_rule(PFCWD_TABLE_NAME, PFCWD_RULE_NAME_1, config_qualifiers, action="DROP")
+            time.sleep(5)
+            rule_oid = self.find_acl_rule_oid(dvs)
+            dvs_acl.verify_acl_rule(expected_sai_qualifiers, action="DROP", acl_rule_id=rule_oid)
```

Comment:
same feedback as below w.r.t. sleep + verify_acl_rule

Reply:
Fixed verify_acl_rule but without sleep it seems to be failing.